plugin/DirDiff.vim

```diff
@@ -616,12 +621,22 @@ function! <SID>CloseDiffWindows()
         wincmd k
         " Ask the user to save if buffer is modified
         call <SID>AskIfModified()
-        bd!
+        if g:DirDiffBufferDelete
+            bd!
+        else
+            diffoff
+            q
```

Comment:
Not good to assume recursive map. Should call `:call s:DirDiffQuit()` explicitly.